src/test/java/org/terasology/itempipes/ItemPipesTest.java

```diff
@@ -163,43 +185,38 @@ private byte getConn(Vector3i location) {
     /**
      * Deals damage to block on given location (simulates the situation when player destroys a block)
      *
-     * @param location     location of the block to deal damage
+     * @param location location of the block to deal damage
      * @param damageAmount amount of the damage to be dealt.
      */
-    private EntityRef dealDamageOn(Vector3i location, int damageAmount) {
+    private EntityRef dealDamageOn(Vector3ic location, int damageAmount) {
         EntityRef block = blockEntityRegistry.getBlockEntityAt(location);
         block.send(new DoDamageEvent(damageAmount, EngineDamageTypes.DIRECT.get()));
 
         return block;
     }
 
-    /**
-     * Simulates the situation when a player places a block.
-     *
-     * @param location location of the block to be placed.
-     * @param id       ID of the block family.
-     */
-    private EntityRef placeBlock(Vector3i location, String id) {
-        forceAndWaitForGeneration(location);
-
-        BlockItemFactory blockItemFactory = new BlockItemFactory(entityManager);
-        BlockFamily family = blockManager.getBlockFamily(id);
-        EntityRef newBlock = blockItemFactory.newInstance(family);
+    private void placePipe(Vector3ic location) {
+        helper.forceAndWaitForGeneration(JomlUtil.from(location));
 
-        Block block = family.getBlockForPlacement(location, Side.TOP, Side.FRONT);
+        worldProvider.setBlock(location,
+            itemPipesBlockFamily.getBlockForPlacement(new BlockPlacementData(location, Side.FRONT,
+                new org.joml.Vector3f())));
```

Comment:
in contrast, here's a place where there's a `new Vector3f()` and I'm guessing its value _doesn't_ matter to any of the assertions.